motorApp/SM300Src/SM300Driver.h

```diff
@@ -0,0 +1,69 @@
+/*
+Motor driver support for the SM300 controller.
+*/
+
+
+// controller-specific parameters yet
+#define NUM_SM300_PARAMS 4
```

Comment:
The ususal asyn way is not to hard code the number of parameters, but to define the first and last parameter names and then use "last - first + 1" so new parameters can be added without needing to (or forgetting to) update the number. In  asyn 4.32 the "paramTableSize" argument is deprecated, but i expect this has not made it into asynMotorController yet